nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/BulkRemoveEnrichmentAction.java

```diff
@@ -94,8 +94,9 @@ public void startBucket(String bucketKey) {
         }
 
         @Override
-        protected void compute(CoreSession session, List<String> ids, Map<String, Serializable> properties) {
+        protected void compute(CoreSession session, List<String> listOfIds, Map<String, Serializable> properties) {
```

Comment:
you didnt rename anything in the method, is it ok?

Reply:
ah, I was debugging something for the. streams in the meantime. Will reset it. thanks for the catch